src/app/(private)/(routes)/(member)/user/[userName]/components/Header.tsx

```diff
@@ -38,12 +39,20 @@ const Header = ({
     }
   }, [currentUser.id, followers])
 
+  useEffect(() => {}, [scoreboard?.data.points])
+
   const handleFollowToggle = async () => {
-    if (isFollowed) {
-      await unfollow(user.id)
-      return
+    try {
+      if (isFollowed) {
+        await unfollow(user.id)
+      } else {
+        await follow(user.id)
+      }
+
+      await refetchScoreboard()
+    } catch (error) {
+      console.error('Error during follow/unfollow operation:', error)
```

Comment:
In my opinion, you can remove the console error since the client can't do much about it.